src/main/java/frc/robot/controls/GameControllerDriverConfig.java

```diff
@@ -33,10 +34,12 @@ public void configureControls() {
     // set the wheels to X
     kDriver.get(Button.X).whileTrue(new SetFormationX(super.getDrivetrain()));
     // Enable state deadband after setting formation to X
-    kDriver.get(Button.X).onFalse(new InstantCommand(() -> getDrivetrain().setStateDeadband(true)));
+    kDriver.get(Button.X).onFalse(new InstantCommand(()->getDrivetrain().setStateDeadband(true)));
 
-    // Resets the modules to absolute if they are having the unresolved zeroing
-    // error
+    // Run SysID commands
+    kDriver.get(Button.Y).onTrue(new SysIDDriveCommand(getDrivetrain()));
+    
```

Comment:
We probably don't want this to be an easily accessible button on our final controls, in case it's accidentally pressed during a match. Maybe move it to DPad up or something else nowhere near any other buttons.

Reply:
removed it